2018-03-06T11:05:50+01:00	[WARN]	[[Questions::writeAnswer()] Otazka ma uvedeny jiny typ, nez byl dorucen v odpovedi na ni. Typ otazky: r, typ odpovedi: t, cislo otazky: 10, ID otazky: 9.
PHP Stack trace:
#0 --(--): Phc_ErrorLog->_errorHandler()
#1 /storage/web/www.nextforce.cz/www/gdpr/library/Phc/ErrorLog.php(101): trigger_error()
#2 /storage/web/www.nextforce.cz/www/gdpr/models/Questions.php(517): Phc_ErrorLog->warning()
#3 /storage/web/www.nextforce.cz/www/gdpr/library/Ibulletin/Content/Indetail.php(919): Questions->writeAnswer()
#4 /storage/web/www.nextforce.cz/www/gdpr/controllers/ServiceController.php(52): Ibulletin_Content_Indetail->saveService()
#5 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Action.php(516): ServiceController->doAction()
#6 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch()
#7 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch()
#8 /storage/web/www.nextforce.cz/www/gdpr/index.php(155): Zend_Controller_Front->dispatch()
 in /storage/web/www.nextforce.cz/www/gdpr/library/Phc/ErrorLog.php on line 101]	[/gdpr/service/1/save/pageviewsid/2654/]	[Mozilla/5.0 (Windows NT 6.1; Win64; x64; Trident/7.0; rv:11.0) like Gecko]	[Inbox v3.8 r4082]

2018-03-06T18:01:29+01:00	[ERR]	[[Questions::writeAnswer()] Nepodarilo se vytvorit novou odpoved v tabulce answers. answer_num = '4207157494', question_id = '9'. Puvodni vyjimka:
exception 'PDOException' with message 'SQLSTATE[22003]: Numeric value out of range: 7 ERROR:  value "4207157494" is out of range for type integer' in /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Statement/Pdo.php:228
Stack trace:
#0 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Statement/Pdo.php(228): PDOStatement->execute(Array)
#1 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#2 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Abstract.php(479): Zend_Db_Statement->execute(Array)
#3 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO "an...', Array)
#4 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Abstract.php(592): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO "an...', Array)
#5 /storage/web/www.nextforce.cz/www/gdpr/models/Questions.php(547): Zend_Db_Adapter_Abstract->insert('answers', Array)
#6 /storage/web/www.nextforce.cz/www/gdpr/library/Ibulletin/Content/Indetail.php(919): Questions->writeAnswer('10', '+4207157494', 'r', NULL, 194, '1', NULL, NULL, '2793')
#7 /storage/web/www.nextforce.cz/www/gdpr/controllers/ServiceController.php(52): Ibulletin_Content_Indetail->saveService(Object(Zend_Controller_Request_Http))
#8 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Action.php(516): ServiceController->doAction()
#9 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch('doAction')
#10 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 /storage/web/www.nextforce.cz/www/gdpr/index.php(155): Zend_Controller_Front->dispatch()
#12 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[22003]: Numeric value out of range: 7 ERROR:  value "4207157494" is out of range for type integer' in /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Statement/Pdo.php:234
Stack trace:
#0 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Statement.php(300): Zend_Db_Statement_Pdo->_execute(Array)
#1 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Abstract.php(479): Zend_Db_Statement->execute(Array)
#2 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('INSERT INTO "an...', Array)
#3 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Db/Adapter/Abstract.php(592): Zend_Db_Adapter_Pdo_Abstract->query('INSERT INTO "an...', Array)
#4 /storage/web/www.nextforce.cz/www/gdpr/models/Questions.php(547): Zend_Db_Adapter_Abstract->insert('answers', Array)
#5 /storage/web/www.nextforce.cz/www/gdpr/library/Ibulletin/Content/Indetail.php(919): Questions->writeAnswer('10', '+4207157494', 'r', NULL, 194, '1', NULL, NULL, '2793')
#6 /storage/web/www.nextforce.cz/www/gdpr/controllers/ServiceController.php(52): Ibulletin_Content_Indetail->saveService(Object(Zend_Controller_Request_Http))
#7 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Action.php(516): ServiceController->doAction()
#8 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch('doAction')
#9 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /storage/web/www.nextforce.cz/www/gdpr/index.php(155): Zend_Controller_Front->dispatch()
#11 {main}
PHP Stack trace:
#0 --(--): Phc_ErrorLog->_errorHandler()
#1 /storage/web/www.nextforce.cz/www/gdpr/library/Phc/ErrorLog.php(117): trigger_error()
#2 /storage/web/www.nextforce.cz/www/gdpr/models/Questions.php(553): Phc_ErrorLog->error()
#3 /storage/web/www.nextforce.cz/www/gdpr/library/Ibulletin/Content/Indetail.php(919): Questions->writeAnswer()
#4 /storage/web/www.nextforce.cz/www/gdpr/controllers/ServiceController.php(52): Ibulletin_Content_Indetail->saveService()
#5 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Action.php(516): ServiceController->doAction()
#6 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch()
#7 /storage/web/www.nextforce.cz/www/gdpr/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch()
#8 /storage/web/www.nextforce.cz/www/gdpr/index.php(155): Zend_Controller_Front->dispatch()
 in /storage/web/www.nextforce.cz/www/gdpr/library/Phc/ErrorLog.php on line 117]	[/gdpr/service/1/save/pageviewsid/2793/]	[Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko]	[Inbox v3.8 r4082]

